firmware/lib/Gantry/Axis.cpp

```diff
@@ -204,18 +226,22 @@ void isr_limit_switch_y()
     uint32_t interrupt_time = millis();
     // If interrupts come faster than 10ms, assume it's a bounce and ignore
     if (interrupt_time - last_interrupt_time > 50) {
```

Comment:
Do we mean 10ms or 50ms in this case? Is this still a debounce functionality?

Reply:
@oplegion ?

Reply:
Will address in future PR